app/representers/api/v1/unclaimed_user_representer.rb

```diff
@@ -0,0 +1,13 @@
+module Api::V1
+  class UnclaimedUserRepresenter < Roar::Decorator
```

Comment:
@nathanstitt I think this whole thread has been addressed right? (i.e. we're just returning an ID now so there's no data going back to the application?)